types/govuk-frontend/dist/govuk/init.d.ts

```diff
@@ -82,12 +114,12 @@ export function initAll(
  *
  * @template {CompatibleClass} T
  * @param {T} Component - class of the component to create
- * @param {T["defaults"]} [config] - config for the component
- * @param {Element|Document} [$scope] - scope of the document to search within
+ * @param {T["defaults"]} [config] - Config supplied to component
+ * @param {OnErrorCallback<T> | Element | Document | CreateAllOptions<T> } [createAllOptions] - options for createAll including scope of the document to search within and callback function if error throw by component on init
```

Comment:
@romaricpascal Can I flag this as an accidental docs change?

The types for `$scope` include `undefined` but you've documented it as `null` in [**Initialise only on part of a page**](https://frontend.design-system.service.gov.uk/building-your-own-javascript-components/#initialise-only-on-part-of-a-page) with this example:

```js
// If the component does not need any configuration, pass `null` for the configuration
createAll(ProjectComponent, null, $element)
```

Reply:
To clarify, if we'd want to accept `null`, we'd need line 117 for `config` to be the following, is that it?

```js
 * @param {T["defaults"] | null} [config] - Config supplied to component
```

Reply:
Yeah, except all of these types are generated from GOV.UK Frontend source

E.g. You'll need to update [**src/govuk/init.mjs**#L70](https://github.com/alphagov/govuk-frontend/blob/3cbc709d489e04a468d9032a942f26870af656d2/packages/govuk-frontend/src/govuk/init.mjs#L70) in a future release and I can generate again

There's also using `?` to make things nullable like `{?T["defaults"]}`